service/worker/scanner/executions/types.go

```diff
@@ -102,6 +104,10 @@ func (st ScanType) ToInvariants(collections []invariant.Collection) []InvariantF
 				fns = append(fns, invariant.NewInactiveDomainExists)
 			case invariant.CollectionHistory:
 				fns = append(fns, invariant.NewHistoryExists)
+			case invariant.CollectionStale:
```

Comment:
Digging around and looking at what's deployed with flipr keys like `worker.concreteExecutionFixerEnabled`, I think this will still execute as part of the `ConcreteExecutionType` category, making it still a bit dangerous. I don't there's currently a more granular flipr configuration. 

If you want to roll this out separately, I think you need to find a way to introduce a new category similar to `worker.concreteExecutionFixerEnabled`, unless I'm missing something (could easily be wrong, this code always confuses me)

Reply:
yea, I'll explore that and update somehow.  I'm not sure either.

Reply:
Got a concrete answer for ya!  https://github.com/uber/cadence/pull/5361/files#diff-5bdda90b33fbf138bc464971e79ca94b938e3ef2461b9f4dcceddc3088fff7f5R73-R117

Reply:
oh god, I am still not sure I undrestand

Reply:
np, I'm gonna reword to put config as a separate and clearer thing.

basically:
- config controls what invariants get into ^ the list of invariants passed in here, because the enabled config names are converted to an enum lookup which produces this list
- these are filtered by type (which should always match what are passed in, but this is not checked)
- so if you don't have a scanner enabled, its config-key doesn't exist, so the enum isn't found, so it isn't in this list, so the invariant doesn't run in scanner.

which is, like, bleh, but it does work.

Reply:
for "fixer enabled" config: yeah, I think you may be right.  since right now it runs all invariants in fixer :\  (see other comment)